Source/Core/DolphinQt/Main.cpp

```diff
@@ -143,7 +143,7 @@ int main(int argc, char* argv[])
   // code, which makes mouse inputs work again.
   // For more information: https://bugs.dolphin-emu.org/issues/12913
 #if (QT_VERSION >= QT_VERSION_CHECK(6, 3, 0))
-  putenv("QT_XCB_NO_XI2=1");
+  setenv("QT_XCB_NO_XI2", "1", true);
```

Comment:
Huh, I didn't know about `putenv` but [apparently](https://www.man7.org/linux/man-pages/man3/putenv.3.html) "The string pointed to by string becomes part of the environment, so altering the string changes the environment." - so that's why it'd take a mutable string. That seems like really weird design and [apparently it's not recommended anymore](https://stackoverflow.com/a/5876818).